### ISSUE TITLE
Out of memory lock-up

### ISSUE THREAD

**Issue by JasY95** [Tom94/tev#125]
A useful feature would be to choose whether to load the images into memory, for particularly large EXR files. Loading a sequence of 600 130MB caused the system to entirely lock up.

**Comment by Tom94** [comment]
Agreed! Unfortunately, this requires a few non-trivial changes throughout tev, so I can't say when I'll get to it, but I'm putting it high on the todo list.
